### PR TITLE
feature/error_page

### PR DESCRIPTION
**Pull request by StepanPotiienko**

### CODE REVIEW

File: app/error.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    <div className="w-screen h-full">
+      <Header />
+      <div className="h-screen">
+        <div className="flex flex-row h-full max-[1280px]:flex-col">
+          <div className="w-full flex flex-row items-center max-[1280px]:h-screen">
+            <ErrorMessage />
+          </div>
+          <div className="w-1/2 h-full flex flex-col self-end items-end justify-end max-[1280px]:w-3/4">
+            <Image
+              src={'/images/sad_cat.png'}
```
Comment (Markusplay) [inline app/error.tsx]: you don't need curly braces here

File: .json
Comment (Markusplay) [inline .json:1]: Delete this file

**Review by Markusplay — CHANGES_REQUESTED** [review]

File: app/error.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        </li>
+        <li className="w-full flex justify-center items-center mt-6 max-[1280px]:mt-3">
+          <Link href="/" className="w-1/2 max-[1280px]:w-3/4">
+            <Button className="min-[389px]:text-[16px] w-full">Головна</Button>
+          </Link>
+        </li>
+      </ul>
+    </div>
+  );
+};
```
Comment (Markusplay) [inline app/error.tsx:60]: You don't need to take it to a component. Make all the code in one return

**Review by Markusplay — CHANGES_REQUESTED** [review]

File: public/images/sad_cat.png
Comment (Markusplay) [inline public/images/sad_cat.png:1]: Use HD Gif instead of image

**Review by Markusplay — CHANGES_REQUESTED** [review]

**Review by Markusplay — CHANGES_REQUESTED** [review]

File: app/error.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+import Link from 'next/link';
+
+import { Button } from '@/components/ui/button';
+
+export default function ErrorPage() {
+  return (
+    <div className="w-screen h-full">
+      <div className="h-screen">
+        <div className="flex flex-row h-screen max-[1280px]:flex-col">
+          <div className="w-full flex flex-row items-center max-[1280px]:h-screen">
```
[the remark this answers is not in the document]
Comment (StepanPotiienko) [reply app/error.tsx:13]: We do, because later we add "flex-col" for mobile devices.

**Review by Markusplay — APPROVED** [review]

**Pull request merged**